mappings/publisher_backlist.py

```diff
@@ -0,0 +1,103 @@
+from .json import JSONMapping
+
+class PublisherBacklistMapping(JSONMapping):
+    def __init__(self, source):
+        super().__init__(source, {})
+        self.mapping = self.createMapping()
+
+    def createMapping(self):
+        return {
+            'title': ('Title', '{0}'),
+            'authors': ('Author(s)', '{0}'),
+            'dates': [('Pub Date', '{0}|publication_date')],
+            'publisher': [('Publisher (from Projects)', '{0}||')],
+            'identifiers': [
+                ('ISBN', '{0}|isbn'),
+                ('OCLC', '{0}|oclc')
+            ],
+            'rights': ('DRB Rights Classification', '{0}||||'),
+            'contributors': [('Contributors', '{0}|||contributor')],
+            'subjects': ('Subject 1', '{0}'),
+            'source': ('Projects', '{0}'),
+            'publisher_project_source': ('Publisher (from Projects)', '{0}')
+        }
+
+    def applyFormatting(self):
+        self.record.has_part = []
+        if self.record.source:
+            source_list = self.record.source.split(' ')
+            self.record.source = source_list[0]
+
+        if self.record.publisher_project_source:
+            publisher_source = self.record.publisher_project_source[0]
+            self.record.publisher_project_source = publisher_source
+
+        if self.record.authors:
+            self.record.authors = self.format_authors()
+
+        if self.record.subjects:
+            self.record.subjects = self.format_subjects()
+
+        if self.record.identifiers:
+            if len(self.record.identifiers) == 1:
+                source_id = self.record.identifiers[0].split('|')[0]
+                self.record.source_id = f'{self.record.source}_{source_id}'
+                self.record.identifiers = self.format_identifiers()
+            else:
+                source_id = self.record.identifiers[1].split('|')[0]
+                self.record.source_id = f'{self.record.source}_{source_id}'
+                self.record.identifiers = self.format_identifiers()
+
+        self.record.rights = self.format_rights()
+
+    def format_authors(self):
+        authorList = []
+
+        if ';' in self.record.authors:
+            authorList = self.record.authors.split('; ')
+            newAuthorList = [f'{author}|||true' for author in authorList] 
```

Comment:
Should there be 2 or 3 pipes. I thought I've seen other author lists have only 2 pipes